src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -43,7 +46,7 @@ export default class GeFormRenderer extends NavigationMixin(LightningElement) {
     @api donorApiName;
     @api donorRecord;
 
-    fieldNames = [];
+    fieldNames = [ ACCOUNT_NAME_FIELD, CONTACT_NAME_FIELD ];
```

Comment:
In getRecordFieldNames in utilTemplateBuilder.js we initialize the array of fields to 
let fieldNames = [`${apiName}.Id`]; 
to handle the case when the template does not contain account or contact field names and the form was not rendering. Can we get rid of that line now that we initialize the array with Contact.Name and Account.Name?

Reply:
I'm working in another branch and the default template loads fine for me from contact/account. This bug was already fixed once.

Reply:
This bug was already fixed once, and I don't see it happening in my branch. Did we regress maybe at some point?